### ISSUE TITLE
Rupture not working out of the box

### ISSUE THREAD

**Issue by WoodyWoodsta** [juliancwirko/s-grid#14]
Is this something that we need to import manually? Perhaps this could be slightly clearer in the docs!

**Comment by juliancwirko** [comment]
Yes, you're right. Because here we have only s-grid library which can be used with for example Rupture and other npm packages.

**Comment by WoodyWoodsta** [comment]
Ok understood. I was able to implement the examples using rupture after importing it manually. As mentioned, it would be great if the docs included the fact that s-grid does not bundle in rupture.

Otherwise, great work :)
